packages/fxa-auth-server/test/remote/subscription_tests.js

```diff
@@ -66,8 +66,11 @@ describe('remote subscriptions:', function () {
 
     beforeEach(async () => {
       client = await clientFactory.create(config.publicUrl, server.uniqueEmail(), 'wibble');
-      defaultRefreshToken = mockRefreshToken(CLIENT_ID_FOR_DEFAULT, client.uid, 'profile:subscriptions');
-      refreshToken = mockRefreshToken(CLIENT_ID, client.uid, 'profile:subscriptions');
+      tokens = [
```

Comment:
I got tired of trying to think up appropriate meaningful names for these tokens, hopefully array indexing doesn't harm the readability of the tests too much.